### DISCUSSION TITLE
More details about the parameters used in DenseAmpcor

### DISCUSSION THREAD

**Discussion by riyasmjgeo** [isce-framework/isce2#533] [Q&A]
Hello All,

I am using DenseAmpcor for offset tracking measurement. I would like to know more about the parameters used in the process. I can see comments in the DenseAmpcor.py file, which is really useful. However, is there any documentation, article, or any medium that explains them in a bit more detailed manner? Pls let me know if anyone knows any other sources.

Thank you!

**Comment by fdelgadodelapuente** [comment]
https://github.com/parosen/Geo-SInC/blob/main/UNAVCO2021/4.4_Offset_stack_for_velocity_dynamics/nb_topsApp_offsets.ipynb

**Reply by riyasmjgeo** [reply]
Thank you @fdelgadodelapuente

**Comment by riyasmjgeo** [comment]
Hello All,

Thank you for helping me so far. I have some particular doubts about the offset pixel tracking technique in ISCE2. 

1. While using `stackSentinel.py`, how do I assign parameters related to offset tracking, such as _search window width_? I figured out that I can specify additional parameters in individual _config_denseOffset_ run files. Instead of modifying each run file, is there any option to assign them while calling the `stackSentinel.py` itself or any other way?

2. Can I use GRD products instead of SLC for offset pixel track in `topsApp.py` or `stackSentinel.py`?

3. Consider using a Sentinel-1 SLC IW pair of 12 days difference for offset pixel tracking of a glacier having a maximum velocity of 10 meters/day. Hence the maximum expected velocity will be 120 meters between the image pair. Considering the resolution of 14.1x 2.3 (Az x Rg), I am assuming the parameters, 
Ampcor search window width = (120/2.3) * 2 = ~100 
Ampcor search window height = (120/14.1) * 2 = ~20
Ampcor window width = 40
Ampcor window height = 8
Skip width = Skip height = 0

What are the conceptual mistakes I am making here? 
What is the importance of the zoom window?
Any suggestions for improvement?

4. Using low window dimensions can result in a noisy outcome. In that case, I can opt for a larger window dimension. That will consequently make me **increase the search window dimensions** (for example, 200width and 50 height) as well. On the other hand, if I know the velocity and I don't want the program to look for a correlation beyond a limit (for example, 100, 25), how do I comprehend both these conditions if they conflict?

**Reply by pburgi-usgs** [reply]
I would also like to know the importance of the zoom window, and also how to change the default value in the xml file.
